code/datums/status_effects/status_effect.dm

```diff
@@ -12,7 +12,7 @@
 	var/alert_type = /obj/screen/alert/status_effect //the alert thrown by the status effect, contains name and description
 	var/obj/screen/alert/status_effect/linked_alert = null //the alert itself, if it exists
 
-/datum/status_effect/New(mob/living/new_owner)
+/datum/status_effect/New(mob/living/new_owner, terrifying_second_argument)
```

Comment:
This is very bad but I couldn't figure out how to do arbitrary arguments like Initialize() does.